project-koku/masu
# Fixing OCP-on-AWS summary table removal

Addresses #440 

**Testing**
1. Setup providers and ingest OCP-on-AWS data
2. Sample the openshift/infrastructures/aws endpoint
3. Re-sample /openshift/infrastructures/aws endpoint, verify data is gone.
4. Re-add OCP account and ingest data
5. Re-sample openshift/infrastructures/aws endpoint, verify data is present
6. Remove the AWS provider
7. Re-sample /openshift/infrastructures/aws endpoint, verify data is gone.

**Test Results**
[masu_issue_440_ut.txt](https://github.com/project-koku/masu/files/3174737/masu_issue_440_ut.txt)
